apps/shell/src/i18n/en.json

```
@@ -42,7 +42,9 @@
     "undelegate": "Undelegate",
     "redelegate": "Redelegate"
   },
-  "governance": {},
+  "governance": {
+    "your-balance": "Your balance: {{localedBalance}} {{localedSymbolUpperCase}}"
```

at first change please this
```suggestion
    "your-balance": "Your balance: {{balance}} {{symbol}}"
```

at second where is the rest of strings?